### PR TITLE
rclone 1.37

### PR DESCRIPTION
**Pull request by ilovezfs**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/rclone.rb [hunk 1]
```diff
@@ -1,10 +1,8 @@
-require "language/go"
-
 class Rclone < Formula
   desc "Rsync for cloud storage"
   homepage "https://rclone.org/"
-  url "https://github.com/ncw/rclone/archive/v1.36.tar.gz"
-  sha256 "a573b70e3aeb355b943dddd6ae9375386fc21bf12dfba601d8d7280f97c4c884"
+  url "https://github.com/ncw/rclone/archive/v1.37.tar.gz"
+  sha256 "a74e284d2368f6fb8e4ac654a31b9b1328ac6078acd3446c9a892cd4bcbe8660"
```
Comment (zmwangx) [inline Formula/rclone.rb]: Mind adding

```
head "https://github.com/ncw/rclone.git"
```

I need the head version, which could be squeezed into this PR.

File: Formula/rclone.rb [hunk 2]
```diff
@@ -15,151 +13,10 @@ class Rclone < Formula
...
-    url "https://github.com/shurcooL/sanitized_anchor_name.git",
-        :revision => "10ef21a441db47d8b13ebcc5fd2310f636973c77"
-  end
-
   def install
     ENV["GOPATH"] = buildpath
     mkdir_p buildpath/"src/github.com/ncw/"
     ln_s buildpath, buildpath/"src/github.com/ncw/rclone"
-    Language::Go.stage_deps resources, buildpath/"src"
     system "go", "build", "-o", bin/"rclone"
```
Comment (zmwangx) [inline Formula/rclone.rb]: There's also an `rclone.1` in the root directory that could be installed.

**Comment by ilovezfs** [comment]
@zmwangx good?

**Comment by zmwangx** [comment]
👍

**Comment by ilovezfs** [comment]
:tornado:

**Pull request closed without merging**